deployment/Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM debian:jessie
+
+MAINTAINER Tobias Blaser <tblaser@hsr.ch>
+
+# Update system
+RUN apt-get update
+RUN apt-get -y autoremove
+RUN apt-get -y install apt-utils
+
+# Install python 3
+RUN apt-get -y install tar python3 python3-doc python3-setuptools python3-pip python-dev libpq-dev
+RUN pip3 install virtualenv
+
+# Install GEO libraries
+RUN apt-get -y install binutils libgeos-3.4. # GEOS
+RUN apt-get -y install libproj0 # PROJ.4
+RUN apt-get -y install python-gdal # GDAL
+
+# Install database
+RUN apt-get -y install postgresql postgresql-client
+RUN apt-get -y install postgis osmctools postgresql-9.4-postgis-2.1
+RUN apt-get -y install python3-psycopg2
+RUN service postgresql start
+
+# Install apache
+RUN apt-get -y install apache2 apache2-utils libapache2-mod-wsgi-py3
+RUN a2enmod wsgi
+RUN service apache2 restart
+
+USER postgres
+    # Fix pg template ascii bug
+    # @source: http://stackoverflow.com/questions/16736891/pgerror-error-new-encoding-utf8-is-incompatible
+    RUN service postgresql start &&\
+        psql -c "UPDATE pg_database SET datistemplate = FALSE WHERE datname = 'template1';" &&\
+        psql -c "DROP DATABASE template1;" &&\
+        psql -c "CREATE DATABASE template1 WITH TEMPLATE = template0 ENCODING = 'UNICODE';" &&\
+        psql -c "UPDATE pg_database SET datistemplate = TRUE WHERE datname = 'template1';" &&\
+        psql -c "\c template1" &&\
+        psql -c "VACUUM FREEZE;"
```

Comment:
Docker runs only changed commands if you rebuild a container. If you move it to a script, docker can not detect changes. 
@das-g If you think it's not a problem, we can try to move it (Needs testing again).